test/integration/goldens/kms/KeyManagementServiceStubSettings.java

```diff
@@ -1301,6 +1290,17 @@ public Builder applyToAllUnaryMethods(
       return getLocationSettings;
     }
 
+    /** Returns the builder for the settings used for calls to setIamPolicy. */
+    public UnaryCallSettings.Builder<SetIamPolicyRequest, Policy> setIamPolicySettings() {
+      return setIamPolicySettings;
+    }
+
+    /** Returns the builder for the settings used for calls to testIamPermissions. */
+    public UnaryCallSettings.Builder<TestIamPermissionsRequest, TestIamPermissionsResponse>
+        testIamPermissionsSettings() {
+      return testIamPermissionsSettings;
+    }
+
```

Comment:
non-blocking comment: without debugging, I'm not seeing the cause for why these methods are moved in the generated code.

Reply:
To avoid arbitrary reordering, I'll sort everything in alphabetical order (except for mixin APIs). Adding this to a separate PR, so this diff should go away then. Thanks!